Week3/homework/step2-3.js

```diff
@@ -4,41 +4,42 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  for (let i = 0; i < num; i++) {
+    result = result.concat(str);
```

Comment:
`concat` is a good solution, but this could be simpler. Try just using regular string concatenation, ie; `result += str`